The currently activated Python version 2.7.16 is not supported by the project (^3.5).
Trying to find and use a compatible version. 
Using python3 (3.7.6)
<a name="ascii_art"></a>
# ascii\_art

<a name="ascii_art.weigh"></a>
#### weigh

```python
weigh(c, font)
```

Find and return density of dark pixels in the given character

<a name="ascii_art.find_char"></a>
#### find\_char

```python
find_char(pixel, weighted_chars)
```

Find and return character from dict with similar darkness to image pixel

<a name="ascii_art.image_to_ascii"></a>
#### image\_to\_ascii

```python
image_to_ascii(image, width=None, height=None, chars=' .\',:;+*?%S#@', font=None, invert=False, normalize=False)
```

Convert image to ASCII chars

<a name="ascii_art.test"></a>
# ascii\_art.test

